### PR TITLE
Add plot for homozygosity per chromosome query.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

**Comment by mbookman** [comment]
LGTM

File: R/PlatinumGenomes-QC/Sample-Level-QC.Rmd [hunk 1]
```diff
@@ -251,6 +252,35 @@ Let's accumulate our sample-specific results for later use.
...
+joinedResult <- inner_join(result, sampleInfo)
+```
+
+And visualizing the results:
+```{r homozygosityAltSummary, fig.align="center", fig.width=12, message=FALSE, comment=NA}
+  ggplot(joinedResult, aes(y=F, x=reference_name, color=sex)) +
+  geom_boxplot() +
+  facet_grid(sex ~ .) +
+  scale_y_continuous(labels=comma) +
+  ylab("Fraction of Homozyous Variants") +
```
Comment (pgrosu) [inline R/PlatinumGenomes-QC/Sample-Level-QC.Rmd]: s/Homozyous/Homozygous

**Comment by pgrosu** [comment]
Does this error in the Travis build matter?  It is at the end of the [raw Travis build log file](https://s3.amazonaws.com/archive.travis-ci.org/jobs/107831364/log.txt):

```Bash
Downloaded: https://repo.maven.apache.org/maven2/org/eclipse/jgit/org.eclipse.jgit/3.2.0.201312181205-r/org.eclipse.jgit-3.2.0.201312181205-r.jar (1811 KB at 6073.8 KB/sec)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 14.542 s
[INFO] Finished at: 2016-02-08T18:08:19+00:00
[INFO] Final Memory: 33M/228M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eluder.coveralls:coveralls-maven-plugin:2.2.0:cobertura (default-cli) on project non-variant-segment-transformer: Build error: Source encoding must be defined -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

travis_time:end:0db9f5f1:start=1454954883250206324,finish=1454954899471526791,duration=16221320467
[0Ktravis_fold:end:after_success
[0K
Done. Your build exited with 0.
```
~p

**Comment by pgrosu** [comment]
This is just a recommendation.  In the plot I think you are performing the homozygous:heterzygous ratio.  It might be easier to show it flipped around, and probably not including chrM/X/Y for some analysis might help.  Here's a plot of the other way around after averaging by reference:

ratio

~p

**Comment by deflaux** [comment]
Fixed typos and filed https://github.com/googlegenomics/codelabs/issues/51

**Pull request merged**